src/MainWindow.cpp

```diff
@@ -158,39 +160,104 @@ MainWindow::MainWindow(QWidget* parent)
    connect( &(Database::instance()), SIGNAL( deletedSignal(BrewNote*)), this, SLOT( closeBrewNote(BrewNote*)));
+
+   return;
+}
+
+void MainWindow::setSizesInPixelsBasedOnDpi()
+{
+   //
+   // Default icon sizes are fine for low DPI monitors, but need changing on high-DPI systems.
+   //
+   // Fortunately, the icons are already SVGs, so we don't need to do anything more complicated than tell Qt what size
+   // in pixels to render them.
+   //
+   // For the moment, we assume we don't need to change the icon size after set-up.  (In theory, it would be nice
+   // to detect, on a multi-monitor system, whether we have moved from a high DPI to a low DPI screen or vice versa.
+   // See https://doc.qt.io/qt-5/qdesktopwidget.html#screen-geometry for more on this.
+   // But, for now, TBD how important a use case that is.  Perhaps a future enhancement...)
+   //
+   // Low DPI monitors are 72 or 96 DPI typically.  High DPI monitors can be 168 DPI (as reported by logicalDpiX(),
+   // logicalDpiX()).  Default toolbar icon size of 22×22 looks fine on low DPI monitor.  So it seems 1/4-inch is a
+   // good width and height for these icons.  Therefore divide DPI by 4 to get icon size.
+   //
+   auto dpiX = this->logicalDpiX();
+   auto dpiY = this->logicalDpiY();
+   Brewtarget::logD(QString("Logical DPI: %1,%2.  Physical DPI: %3,%4")
+      .arg(dpiX)
+      .arg(dpiY)
+      .arg(this->physicalDpiX())
+      .arg(this->physicalDpiY()));
+   auto defaultToolBarIconSize = this->toolBar->iconSize();
+   Brewtarget::logD(QString("Default toolbar icon size: %1,%2")
+      .arg(defaultToolBarIconSize.width())
+      .arg(defaultToolBarIconSize.height()));
+   this->toolBar->setIconSize(QSize(dpiX/4,dpiY/4));
+
+   //
+   // Historically, tab icon sizes were, by default, smaller (16×16), but it seems more logical for them to be the same
+   // size as the toolbar ones.
+   //
+   auto defaultTabIconSize = this->tabWidget_Trees->iconSize();
+   Brewtarget::logD(QString("Default tab icon size: %1,%2")
+      .arg(defaultTabIconSize.width())
+      .arg(defaultTabIconSize.height()));
+   this->tabWidget_Trees->setIconSize(QSize(dpiX/4,dpiY/4));
+
+   //
+   // Default logo size is 100×30 pixels, which is actually the wrong aspect ratio for the underlying image (currently
+   // 265 × 66 - ie aspect ratio of 4.015:1).
+   //
+   // Setting height to be 1/3 inch seems plausible for the default size, but looks a bit wrong in practice.  Using 1/2
+   // height looks better.  Then width 265/66 × height.  (Note that we actually put the fraction in double literals to
+   // avoid premature rounding.)
+   //
+   // This is a bit more work to implement because its a PNG image in a QLabel object
+   //
+   Brewtarget::logD(QString("Logo default size: %1,%2").arg(this->label_Brewtarget->width()).arg(this->label_Brewtarget->height()));
+   this->label_Brewtarget->setScaledContents(true);
+   this->label_Brewtarget->setFixedSize((265.0/66.0) * dpiX/2, dpiY/2);
```

Comment:
@matty0ung 
I may be out in deep waters here swimming with the big boys :), but as I read this you only scale the X?
Should you not be scaling the Y as well?
i.e.  `this->label_Brewtarget->setFixedSize((265.0/66.0) * dpiX/2, (265.0/66.0) * dpiY/2);`

Reply:
@mattiasmaahl Fair question.  In this instance, the code is (I believe!) correct but my comment could be better.  My intent was that the Y height should be half an inch - hence dpiY/2.  Then the ratio of width to height is 265:66 (from the dimensions of the original image).  So we multiply half an inch (dpiX) by 265/66.  I'll expand the comments a bit.